kpi/deployment_backends/kobocat_backend.py

```diff
@@ -1066,7 +1065,7 @@ def __prepare_bulk_update_payload(cls, request_data: dict) -> dict:
     def __validate_bulk_update_submissions(submissions: list) -> list:
         if len(submissions) == 0:
             raise KobocatBulkUpdateSubmissionsClientException(
-                detail=_('No submissions match the given `submission_ids`')
+                detail='No submissions match the given `submission_ids`'
```

Comment:
I think, this comment is returned to users. We need to keep the translation.
All this code has been removed anyway in  basic case management branch. Would you mind to keep it to avoid conflicts on merge?